extension/src/experiments/columns/collect/util.ts

```diff
@@ -106,17 +109,15 @@ export const mergeAncestors = (
   limitedDepthAncestors: string[],
   join: (...pathArray: string[]) => string
 ) => {
-  if (!acc[path]) {
```

Comment:
[F] I've changed the way that the code is structure to avoid coming into this function if the path has already been collected. I've taken the same approach for `collectColumn` below